modules/core/include/opencv2/core/opengl.hpp

```diff
@@ -537,6 +537,19 @@ CV_EXPORTS void convertToGLTexture2D(InputArray src, Texture2D& texture);
  */
 CV_EXPORTS void convertFromGLTexture2D(const Texture2D& texture, OutputArray dst);
 
+/** @brief Acquire GL Buffer object to process on CL side (convert to UMat).
+@param buffer      - source Buffer object.
+@param dst         - destination UMat.
+@param accessFlags - data access flags (ACCESS_READ|ACCESS_WRITE).
+ */
+CV_EXPORTS void acquireGLBuffer(const Buffer& buffer, UMat& dst, int accessFlags);
```

Comment:
Why is used these names: `acquireGLBuffer` and `releaseGLBuffer`?
- OpenCL uses `map/unmap`
- OpenVX uses `access/commit` (but there are allowed copy operations also)
- `acquire/release` is usually associated with resources locks, and `clEnqueueAcquireGLObjects` actually do it exactly.
But, the main function of implementation is `clCreateFromGLBuffer`, not `clEnqueueAcquireGLObjects`, so name looks very strange.